core/tests/protractor_utils/StoryEditorPage.js

```diff
@@ -273,44 +275,56 @@ var StoryEditorPage = function() {
   };
 
   this.changeStoryTitle = async function(storyTitle) {
-    await storyTitleField.clear();
-    await storyTitleField.sendKeys(storyTitle);
+    await action.clear('Story title Field', storyTitleField);
+    await action.sendKeys(
+      'Story Title Field',
+      storyTitleField, storyTitle);
   };
 
   this.returnToTopic = async function() {
     await general.scrollToTop();
-    await returnToTopicButton.click();
+    await action.click(
+      'Return To Topic Button',
+      returnToTopicButton);
     await waitFor.pageToFullyLoad();
   };
 
   this.changeStoryDescription = async function(storyDescription) {
-    await storyDescriptionField.clear();
-    await storyDescriptionField.sendKeys(storyDescription);
+    await action.clear(
+      'Story Description Field', storyDescriptionField);
+    await action.sendKeys(
+      'Story Description Field',
+      storyDescriptionField, storyDescription);
   };
 
   this.changeStoryNotes = async function(richTextInstructions) {
-    await openStoryNotesEditorButton.click();
+    await action.click(
+      'Open Story Notes Editor Button', openStoryNotesEditorButton);
     var storyNotesEditor = await forms.RichTextEditor(
       notesEditor);
-    await storyNotesEditor.clear();
+    await action.clear('Story Notes Editor', storyNotesEditor);
     await richTextInstructions(storyNotesEditor);
-    await saveStoryNotesEditorButton.click();
+    await action.click(
+      'Save Story Notes Editor Button', saveStoryNotesEditorButton);
   };
 
   this.saveStory = async function(commitMessage) {
     await waitFor.elementToBeClickable(
       saveStoryButton,
       'Save story button takes too long to be clickable');
```

Comment:
We don't need this now that we're using `action.click` below. `action.click` does this waitFor automatically